mkdocs.yml

```diff
@@ -26,14 +26,15 @@ pages:
   - 'Introduction': index.md
   - 'Motivation': motivation.md
   - 'Architecture': architecture.md
-  - 'Getting Started': user-guide.md
-  - 'User Guide':
-    - 'Creating Nix builds for your projects': user-guide/projects.md
-    - 'Stack Projects': user-guide/stack-projects.md
-    - 'Cabal Projects': user-guide/cabal-projects.md
-    - 'Bumping Hackage and Stackage snapshots': user-guide/hackage-stackage.md
-    - 'How to create a development environment': user-guide/development.md
-    - 'How to cross-compile your project': user-guide/cross-compilation.md
+  - 'Tutorials':
+    - 'Getting Started': tutorials/getting-started.md
+    - 'Mapping non-Haskell dependencies to Nixpkgs': dev/pkg-map.md
```

Comment:
I'm not sure this should be in here, or not so prominent. Most people shouldn't have to do this, it's more advanced/troubleshooting.

Reply:
I think it's quite common to hit this (someone is having trouble just now on #haskell.nix, but the tutorial should be expanded), currently it's 4th item in the tutorial list.

Reply:
Also cover https://github.com/input-output-hk/haskell.nix/issues/674